aten/src/ATen/templates/TensorBody.h

```diff
@@ -124,12 +124,16 @@ class TORCH_API Tensor: public TensorBase {
   Tensor conj() const {
     if (!this->is_complex()) {
       return *this;
-    } else {
-      if (this->is_sparse()) {
-        return this->conj_physical();
-      }
+    }
+
+    // Path for dense and nested.
+    // NOTE: conj() is not yet implemented for nested.
+    if (this->layout() == at::kStrided) {
```

Comment:
This should also be used for Mkldnn no?

Reply:
Is it really? Looks like `conj()` is only defined in `TensorBody.h`.

Reply:
I believe it would be `this->is_mkldnn()`

Reply:
MKLDNN does not work with complex dtypes in PyTorch, so it follows the `!is_complex()` path. No action needed.

Reply:
@nikitaved  - but what if it does one day?

The previous behavior just did nothing when you call conj on an unsupported layout. Now it always calls conj_physical with a comment indicating that it's only for sparse and sparse compressed layouts. If that's what we assume, do we want it to error out for any other layout?

Reply:
Sure, I can implant a switch.